qutebrowser/mainwindow/prompt.py

```diff
@@ -616,6 +616,9 @@ def __init__(self, question, parent=None):
         self._init_texts(question)
         self._init_key_label()
 
+        self._expands_user = False
+        self._index_end = 0
```

Comment:
Just realized the `self._index_end` variable probably doesn't need to be a variable in self, just in a function. And even if it does it needs a better name.